Release 0.1 - Device Authorizer with persistent backend (#1)
RADAR-base/RADAR-Rest-Source-Auth: README.md

> # RADAR-Device-Auth

Reviewer: ```suggestion
# RADAR-REST-Source-Auth
```

An application to get authorization from device users to access their data through 3rd party APIs. Mainly supports OAuth2 Authorization code follow to request authorization and processes the returned authorization code to add new entries of users. It creates new users and adds properties compatible and required by the RADAR-REST-CONNECTOR/RADAR-FITBIT-REST-CONNECTOR.

## Features supported
1. It has one active entity where we store user properties.
2. Has liquibase support to enable seamless database schema migration.
3. Has a simple web-service with REST Endpoints to share configured source-type client details and authorized users.
4. Currently various source-types can be configured using a YAML file and these entries are stored in memory.

> ## APIs to be used by Device-Connectors

Reviewer: ```suggestion
## APIs to be used by RADAR REST Connectors
```

> Device `Source-Connectors` can use the APIs as follows.

Reviewer: ```suggestion
RADAR REST Source Connectors can use the APIs as follows.
```
